pkgs/tools/graphics/waifu2x-converter-cpp/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, opencv3, opencl-headers,
+  cudaSupport ? false , cudatoolkit ? null }:
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "waifu2x-converter-cpp";
+  version = "5.2.1";
+
+  src = fetchFromGitHub {
+    owner = "DeadSix27";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1cfab4239zidql7wgn0gprnz0zp6sy2dknxr5pa8vx7c6g6vgh20";
+  };
+
+  preConfigure = ''
+    mkdir -p $out
+    cp -r models_rgb $out/
```

Comment:
According to https://github.com/DeadSix27/waifu2x-converter-cpp/blob/master/BUILDING.md it looks like this step is only required on MacOS/OSX?

Also, why is this in preConfigure instead of postInstall?